src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -75,6 +78,7 @@ internal abstract partial class AbstractNavigateToSearchService
 
                 cancellationToken.ThrowIfCancellationRequested();
                 var declarationInfo = await document.GetSyntaxTreeIndexAsync(cancellationToken).ConfigureAwait(false);
+                var root = default(SyntaxNode);
```

Comment:
I hate this.  Just use `SyntaxNode root = null;`  If you do this, I have to think about whether `SyntaxNode` is a struct or not.